docker/appserver/Tomcat/Dockerfile

```diff
@@ -1,8 +1,16 @@
+ARG ALPINE_VERSION=latest
 ARG TOMCAT_VERSION=8.5.60-jdk8
-FROM tomcat:${TOMCAT_VERSION} AS iaf-as-tomcat
 
+# unpack the framework in a separate image to prevent an additional layer
+# adding the framework already unpacked allows adding properties from the resources directory
```

Comment:
Het is niet nodig om properties in de uitgepakte war te zetten. Ze kunnen ook op het Tomcat classpath gezet worden.
Ik vind het ook niet elegant om de .war uit te pakken.

Reply:
Teruggedraaid, had niet door dat het met op het classpath zetten het ook kon, bij een eerste test had die resources niet gepakt maar lag aan het classpath dat niet helemaal goed stond.